bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYZprime/proc_card_mg5.template

```diff
@@ -0,0 +1,9 @@
+import model_v4 $modelfile
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
```

Comment:
Is the model per default 5FS or 4FS?

Reply:
4FS I believe, maybe @florez could help us to clarify this

Reply:
The model does allow the proton and the jet to be defined with up to 5 flavors. Nevertheless, we do not include the b-jet in the proton and jet definitions, because it produces a negligible increase in the production cross section (< 1%) while it increases computing time significantly.